src/components/PlayerDash.jsx

```diff
@@ -32,30 +32,39 @@ export const PlayerDash = React.createClass({
   },
 
   render() {
-    // console.log(this.state.players);
-    const player = _.find(this.state.players, p => p.id == this.props.params.playerId);
+    let { players, graphData } = this.state;
+
+    const player = _.find(players, p => p.id == this.props.params.playerId);
+
     return (
       <div className="Player">
       { player &&
-          <div className="col-md-3">
-            <PlayerCard {...player} />
-          </div>
```

Comment:
I'll merge as is but I'm gonna indent the `<PlayerCard...` & `<EloGraph...` as soon as I get the chance :wink: